src/DotLiquid/StandardFilters.cs

```diff
@@ -218,14 +234,21 @@ public static string TruncateWords(string input, int words = 15, string truncate
 
         /// <summary>
         /// Split input string into an array of substrings separated by given pattern.
+        ///
+        /// If the pattern is empty the input string is converted to an array of 1-char
+        /// strings (as specified in the Liquid Reverse filter example).
         /// </summary>
         /// <param name="input"></param>
         /// <param name="pattern"></param>
         /// <returns></returns>
         public static string[] Split(string input, string pattern)
         {
-            return input.IsNullOrWhiteSpace()
-                ? new[] { input }
+            if (input.IsNullOrWhiteSpace())
+                return new[] { input };
+
+            // If the pattern is empty convert to an array as specified in the Liquid Reverse filter example.

```

Comment:
of the Shopify specification. There are typically two if not more Shopify pages for each filter, so consider adding the exacting example URL.

Reply:
Added URL to reverse filter.